### ISSUE TITLE
Vet Center name change: Provo Vet Center -> Col. Gail S. Halvorsen 'Candy Bomber' Veterans Center

### ISSUE THREAD

**Issue by stefaniefgray** [department-of-veterans-affairs/va.gov-cms#13835]
## Vet Center, Outstation, Mobile Vet Center facility name change - Description
Vet Centers, Mobile Vet Centers, and Vet Center Oustations may all be subject to name changes in VAST. Not all steps apply to each type of facility -- please pay attention and make sure you've done the relevant steps based on facility type.

## Intake
- [x] What triggered this runbook? (Flag in CMS, Help desk ticket, Product team, VHA Digital Media)
Trigger: Facility flag

- [x] Link to associated help desk ticket (if applicable)
Help desk ticket: https://va-gov.atlassian.net/browse/VAHELP-5865

- [x] Name of submitter (if applicable)
Submitter: @stefaniefgray 

- [ ] If the submitter is an editor, send them links to any relevant KB articles for the Vet Center product. Let them know that facility changes can take between 75 days and 4 months after submitting a request, according to VAST administrators.
KB articles: <insert_kb_article_links>

- [ ] Link to facility in production:
Facility CMS link: https://prod.cms.va.gov/provo-vet-center
Facility API ID: vc_0532V

### CMS help desk steps
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility** with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. **Be sure to preserve the current moderation state of the node when adding or removing the flag.**
- [x] 1. The title (Name of Vet Center field) change comes from Lighthouse to Drupal & is flagged
- [x] 2. If the Vet Center published and is NOT an Outstation/MVC, create a [URL change request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&template=runbook-facility-url-change.md&title=URL+Change+for%3A+%3Cinsert+facility+name%3E), changing the entry from the old facility URL to the new facility URL. (**Note: The URL change request ticket blocks the completion of this ticket.**)

Redirect: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13836

### CMS engineer steps - Vet Center only (not Outstation/MVC)
- [ ] 3. Execute the steps of the URL change request ticket from step 2.

(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)

### Drupal Admin steps (CMS Engineer or Help desk)
_Help desk will complete these steps or escalate to request help from CMS engineering._

**If a Mobile Vet Center or Outstation**
- [ ] 4. Verify which Vet Center it belongs to, and confirm that the "Main Vet Center Location" field is set correctly.

**If a Vet Center**
- [x] 5. In the [Sections taxonomy](https://prod.cms.va.gov/admin/structure/taxonomy/manage/administration/overview), Vet Centers: rename the Section for this Vet Center to match its new name
- [x] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match
- [x] 7. Visit [bulk operations](https://prod.cms.va.gov/admin/content/bulk) page and filter by section = vet center name
- [x] 8. Update URLs for all content in that section by bulk operations
- [x] 9. Resave all content in that section by bulk operations
  
**For all types**
- [x] 10. Edit the Vet Center node by removing flag `Changed name`, and saves the node (with moderation state = published)

#### CMS Help desk (wrap up)
- [ ] 11. Notify editor and any other stakeholders.

**Comment by stefaniefgray** [comment]
Redirect request: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13836

**Comment by stefaniefgray** [comment]
@jilladams @kmariepat-cityfriends Just got editor confirmation that this one and the linked redirect request are good to go!

**Comment by omahane** [comment]
@mmiddaugh We have a redirect in place to https://www.va.gov/col-gail-s-halvorsen-candy-bomber-vet-center created here: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13836

The problem is that I am reading the common name help text and it doesn't fit the change:
>  If the new official name matches the pattern " Vet Center", update the common name to match

The problem is that the URL is based on the common name. So we're wondering whether the Common Name should change, too. If so, should it change to 
Col. Gail S. Halvorsen 'Candy Bomber' Veterans Center
or
Col. Gail S. Halvorsen 'Candy Bomber' Vet Center

Or should we change the redirect to point to 
https://www.va.gov/col-gail-s-halvorsen-candy-bomber-vet**erans**-center
?

**Comment by mmiddaugh** [comment]
Jessica (RCS) confirmed we should retain Provo Vet Center as the common name and URL. 

We'll also need to add the "Also known as" text above the intro (similar to [Prescott Vet Center](va.gov/prescott-vet-center). We might need a separate issue for this @kmariepat-cityfriends @omahane

**Comment by omahane** [comment]
Sounds good. 

The "Also called the" is in this .liquid template:
[content-build/src/site/layouts/vet_center.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/524837be536013fc83717950408e1776d91ab713/src/site/layouts/vet_center.drupal.liquid#L16)

That front-end language should, therefore, be programmatically generated during content build.

**Comment by omahane** [comment]
@stefaniefgray There's nothing else to do on this ticket, so I'm moving to QA.
